packages/graph/hash_graph/lib/interval-ops/src/interval.rs

```diff
@@ -554,15 +553,15 @@ mod tests {
             included_included(5, 15),
             [included_included(5, 10)],
             [excluded_included(0, 15)],
-            [excluded_included(0, 15)],
+            excluded_included(0, 15),
             [excluded_excluded(0, 5)],
         );
         test(
             included_included(5, 15),
             excluded_included(0, 10),
             [included_included(5, 10)],
             [excluded_included(0, 15)],
-            [excluded_included(0, 15)],
+            excluded_included(0, 15),
             [excluded_included(10, 15)],
```

Comment:
Do we want to consider making a struct for these? I appreciate these are tests but it'd be too easy to accidentally swap one of these around.

Reply:
This is just a shorthand for
```rust
IntervalBounds::<u32>::from_bounds(Bound::Excluded(0), Bound::Included(15))
```
to make it more readable

Reply:
Sorry I mean encapsulating the parameters to the actual `test` function

Reply:
I cannot really follow. Could you give an example?

Reply:
```Rust
test(IntervalTestParameters {
    lhs: included_included(5, 15),
    rhs: excluded_included(0, 10),
    intersection: [included_included(5, 10)],
    union: [excluded_included(0, 15)],
    merge: excluded_included(0, 15),
    difference:[excluded_included(10, 15)]
}

Reply:
Oh, yes, that makes sense. I will adjust this as part of #1838